lib/DDG/Spice/Cargo.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Cargo;
+# ABSTRACT: Returns package information from the cargo package manager's registry.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "cargo gcc";
+description "Shows a Cargo package";
+name "Cargo";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Cargo.pm";
+icon_url "/i/crates.io.ico";
+topics "sysadmin", "programming";
+category "programming";
+attribution github  => ['https://github.com/tombebbington', 'Tom Bebbington'],
+            twitter => ['https://twitter.com/tomdbebbington', '@Tom Bebbington'];
+
+triggers startend => 'cargo';
```

Comment:
Fair enough, I'll try to update to be in parallel with npm again
 On 11 Jun 2015 20:10, "Zaahir Moolla" <notifications@github.com> wrote:

> In lib/DDG/Spice/Cargo.pm
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/1852#discussion_r32256594>
> :
>
> > +# ABSTRACT: Returns package information from the cargo package manager's registry.
> > +
> > +use strict;
> > +use DDG::Spice;
> > +
> > +primary_example_queries "cargo gcc";
> > +description "Shows a Cargo package";
> > +name "Cargo";
> > +code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Cargo.pm";
> > +icon_url "/i/crates.io.ico";
> > +topics "sysadmin", "programming";
> > +category "programming";
> > +attribution github  => ['https://github.com/tombebbington', 'Tom Bebbington'],
> > +            twitter => ['https://twitter.com/tomdbebbington', '@Tom Bebbington'];
> > +
> > +triggers startend => 'cargo';
>
> "rust cargo" could be a trigger as well. The Sublime Packages IA uses
> "sublime packages" as th example.
>
> NPM and RubyGems use their names as the triggers, but those are much less
> likely to signal an unrelated query.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duckduckgo/zeroclickinfo-spice/pull/1852/files#r32256594>
> .
>